### PR TITLE
PMM-6969 IA rules API

### PR DESCRIPTION
**Pull request by artemgavrilov**
[PMM-6969](https://jira.percona.com/browse/PMM-6969)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1295

- [ ] https://github.com/Percona-Lab/pmm-api-tests/pull/151
---
- [x] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [x] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Comment by AlekSi** [comment]
There are conflicts now

**Review by AlekSi — CHANGES_REQUESTED** [review]
Merge conflicts

**Review by palash25 — COMMENTED** [review]
few small comments, just skimmed through, will review it once again in some time

**Review by palash25 — APPROVED** [review]

**Comment by artemgavrilov** [comment]
*resolving conflicts

File: Gopkg.toml [hunk 1]
```diff
@@ -14,15 +14,15 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "release/2.13"
```
Comment (AlekSi) [inline Gopkg.toml:17]: PMM-2.0 now

File: Gopkg.toml [hunk 2]
```diff
@@ -14,15 +14,15 @@ required = [
...
 
 [[constraint]]
   name = "github.com/percona-platform/saas"
   branch = "main"
 
 [[constraint]]
   name = "github.com/percona-platform/dbaas-api"
-  branch = "main"
+  branch = "pmm-2.13"
```
Comment (AlekSi) [inline Gopkg.toml:25]: main now

**Review by AlekSi — APPROVED** [review]

**Pull request merged**